dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletFactory.java

```diff
@@ -361,4 +361,10 @@
      */
 	public abstract List<Map<String, String>> getMostViewedContent(String structureInode,Date startDate, Date endDate, User user) throws DotDataException;
 
+    protected List<Contentlet> findPageContentlets(String HTMLPageIdentifier, String containerId, String uniqueId, String orderby,
+            boolean working, long languageId) throws DotDataException, DotStateException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)